scripts/prepare_ontology.py

```diff
@@ -0,0 +1,81 @@
+import sys
+import os
+import yaml
+import json
+import shutil
+
+"""
+python3 scripts/prepare_ontology.py scripts/test/data/data_sources.json fake_ontology
+"""
+
+__NAME = "__NAME__"
```

Comment:
This is slightly confusing -- can you rename the variable something like `PLACEHOLDER` instead?

I would also not use both all caps *and* double underscores for variable names -- all caps is sufficient to differentiate them from other variables in this file.